<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="Wyswietlacz">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>560</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>560</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>560</width>
      <height>200</height>
     </size>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::SegmentStyle::Flat</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <item row="2" column="2">
      <widget class="QPushButton" name="Button_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Button_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Button_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Button_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Button_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Button_0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Button_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Button_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Button_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="Button_DOT">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Button_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Button_ZNAK">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>211</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <item row="5" column="0">
      <widget class="QPushButton" name="Button_percent">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>%</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="Button_plus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="Button_equal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>=</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Button_minus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Button_BACKSPACE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>⌫</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Button_mult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Button_div">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Button_CLEAR">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Outlook</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>C</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="miniwyswietlacz">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::SegmentStyle::Flat</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="O_autorze"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Oautorze_info">
   <property name="text">
    <string>Info.</string>
   </property>
  </action>
  <action name="actionsiema">
   <property name="text">
    <string>siema</string>
   </property>
  </action>
  <action name="O_autorze">
   <property name="text">
    <string>O autorze.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
